src/screens/WalletConnectApprovalSheet.js

```diff
@@ -86,13 +144,69 @@ export default function WalletConnectApprovalSheet() {
     return getDappHostname(dappUrl);
   }, [dappUrl]);
 
+  const approvalAccountInfo = useMemo(() => {
+    const approvalAccountInfo = getAccountProfileInfo(
+      approvalAccount.wallet,
+      walletNames,
+      approvalNetwork,
+      approvalAccount.address
+    );
+    return {
+      ...approvalAccountInfo,
+      accountLabel:
+        approvalAccountInfo.accountENS ||
+        approvalAccountInfo.accountName ||
+        approvalAccount.address,
+    };
+  }, [walletNames, approvalNetwork, approvalAccount]);
+
+  const approvalNetworkInfo = useMemo(() => {
+    const value = get(networkInfo[approvalNetwork], 'value');
```

Comment:
we're trying to get rid of `get` from lodash and use optional chaining instead:
```suggestion
    const value = networkInfo[approvalNetwork]?.value
```

Reply:
cool, I saw it on other places so I guessed that was the way you decided to do it 👌